lib/process-services-cloud/src/lib/app/components/app-list-cloud/app-list-cloud.component.stories.ts

```diff
@@ -1,59 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { applicationConfig, Meta, moduleMetadata, StoryFn } from '@storybook/angular';
-import { ProcessServicesCloudStoryModule } from '../../../testing/process-services-cloud-story.module';
-import { AppListCloudComponent } from './app-list-cloud.component';
```

Comment:
Do we know if HxP still uses that? In ADF there is no usages for that so if they don't use this then it's unused